NLeSC/python-template
# test fix

Ref: #101 

## Description

1. {{...}}/{{...}}/`__version__.py` was simplified. This made the tests succeed locally.
2.  Calls to pytest and sphinx from the tests were changed to fix the problems of running the tests on Github Actions. 
3. One redundant test (test_link) was removed.

## To test

1. install this PR
2. start a relevant virtual environment
3. run `pytest tests`: all 9 tests should succeed
